uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChange.php

```diff
@@ -76,8 +86,29 @@ public static function getBalanceForEntitlement(LeavePeriodEntitlement $periodEn
       LEFT JOIN {$leaveRequestTable} leave_request 
              ON leave_request_date.leave_request_id = leave_request.id AND
                 leave_request.is_deleted = 0
+      LEFT JOIN {$contractTable} contract 
```

Comment:
Can we also use Inner joins for this query just like the one in `getLeaveRequestBalanceForEntitlement` function.